src/pool.rs

```diff
@@ -244,7 +244,31 @@ impl ConnectionPool {
                     let mut servers = Vec::new();
                     let mut replica_number = 0;
 
+                    // Load Mirror settings
                     for (address_index, server) in shard.servers.iter().enumerate() {
+                        let mut mirror_addresses: Vec<Address> = vec![];
```

Comment:
You don't need a type annotation typically, Rust should infer it.

Reply:
Done